main.go

```diff
@@ -232,6 +233,7 @@ The commands work as following:
    ios setlocation [options] [--lat=<lat>] [--lon=<lon>]              Updates the location of the device to the provided by latitude and longitude coordinates. Example: setlocation --lat=40.730610 --lon=-73.935242
    ios setlocationgpx [options] [--gpxfilepath=<gpxfilepath>]         Updates the location of the device based on the data in a GPX file. Example: setlocationgpx --gpxfilepath=/home/username/location.gpx
    ios resetlocation [options]                                        Resets the location of the device to the actual one
+   ios simulatelocation [options] [--lat=<lat>] [--lon=<lon>]	      iOS 17 way to set location of the device to the provided by latitude and longitude coordinates. Example: simulatelocation --lat=55.75582600 --lon=37.61729990 --address=fd19:a731:684f::1 --rsd-port=61935
```

Comment:
Is there a method automatically get address and rsd-port values?

Reply:
If you are using go-ios with `go-ios tunnel start` that's happening implicitly (and you can view the address and port of the managed tunnels with `go-ios tunnel ls`)

Reply:
thanks

Reply:
> If you are using go-ios with `go-ios tunnel start` that's happening implicitly (and you can view the address and port of the managed tunnels with `go-ios tunnel ls`)

thanks

Reply:
> If you are using go-ios with `go-ios tunnel start` that's happening implicitly (and you can view the address and port of the managed tunnels with `go-ios tunnel ls`)

By the way, i'm using mac now, does it work with windows?